### ISSUE TITLE
Periods on initials are not retained

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gdsaxton/python-nameparser#8]
```
Currently

  HumanName("L.I. Han") == HumanName("Li Han")

because

  HumanName("L.I. Han").first == u'LI'

In other words, there are initials which also spell first names.  Retaining and 
normalizing periods on initials would solve this.
```

Original issue reported on code.google.com by `jayqhac...@gmail.com` on 7 Feb 2012 at 7:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `dere...@gmail.com` on 7 Oct 2012 at 10:51
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in v0.2.4
```

Original comment by `dere...@gmail.com` on 11 Feb 2013 at 3:47
- Changed state: **Fixed**
